includes/YesWikiController.php

```diff
@@ -2,13 +2,20 @@
 
 namespace YesWiki\Core;
 
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
 use YesWiki\Core\Service\TemplateEngine;
 use YesWiki\Wiki;
 
 abstract class YesWikiController
 {
     protected $wiki;
+    public $request;
+
+    public function __construct()
+    {
+        $this->request = Request::createFromGlobals();
```

Comment:
Super idée d'ajouter `Request::createFromGlobals`

en termes de performances, j'ai un doute sur l'utilité de mettre le createFromGlobals dans le constructeur de tous les yesWikiController (toutes les actions, tous les handlers, etc, ... ça va en faire des appels).

ne pourrions-nous pas faire un lazyloading `YesWikiController->getRequest()` qui vérifie si `$this->request` est nul et uniquement dans ce cas, fait une requête createFromGlobals

Reply:
les actions et handlers dépendent pas de YesWikiController, mais c'est vrai que tant qu'à faire je peux aussi l'ajouter sur YesWikiPerformeable, et dans ce cas oui faire du lazyloading

Reply:
oups c'est vrai, je me suis avancé trop vite (reste que les yesWikiController sont des services chargés à chaque fois, donc le lazy loading peu faire gagner de précieux millisecondes pour les apis)